src/core/components/auth/authorization-popup.jsx

```diff
@@ -8,6 +8,13 @@ export default class AuthorizationPopup extends React.Component {
     authActions.showDefinitions(false)
   }
 
+  onKeyDown = (e) => {
+    if (e.key === "Escape") {
+      e.stopPropagation();
+      this.close();
```

Comment:
The build is failing because of the semi colons on these two lines.

Reply:
fixed, thanks!